docs/sql-performance-tuning.md

```diff
@@ -40,6 +40,41 @@ Configuration of in-memory caching can be done using the `setConf` method on `Sp
 
 </table>
 
+## Scanning Input Table
+
+Spark SQL can increase the speed while scanning tables via tuning hadoop configurations.
```

Comment:
This at least needs some proofreading, like hadoop -> Hadoop, Max -> max, we scanning -> scanning.
This seems fairly niche and not so SQL related.

Reply:
Proofread. I added it in SQL Tuning because we can set it before executing a SQL. I'd appreciate it if you could give more suggestions.